snap/snapcraft.yaml

```diff
@@ -132,64 +132,67 @@ parts:
       - make
       - zip
   config-seed:
-    source: https://github.com/edgexfoundry/core-config-seed.git
-    source-commit: 79faf75
-    plugin: maven
-    override-build: |
-      snapcraftctl build
-      echo "Installing config files"
+    source: https://github.com/edgexfoundry/core-config-seed-go.git
+    plugin: shell
+    after:
+      - glide
+      - go
+    shell: bash
+    shell-flags: ['-eux', '-o', 'pipefail']
+    shell-command: |
+      export GOROOT="$SNAPDIR/parts/go/install"
```

Comment:
I learned a better trick for this as SNAPDIR went away for me for some reason:
```
export GOROOT=$("$SNAPCRAFT_STAGE/bin/go env GOROOT")
```

Reply:
I think we should also switch all the other instances of using SNAPDIR with the go part to this too